components/azure-kubernetes-service/kafka-client.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: kafkaclient
+spec:
+  replicas: 3
+  serviceName: kafkaclient
+  selector:
+    matchLabels:
+      app: kafkaclient
+  template:
+    metadata:
+      labels:
+        app: kafkaclient
+    spec:
+      containers:
+      - name: kafka
+        image: solsson/kafka:0.11.0.0
```

Comment:
This is a very old release of Kafka, can we use newer?

Reply:
So, I am creating a kafka client inside the cluster because I was following the example form strimzi repo, but It looks like we don't need the client as part of the deployment. Am I right?

Reply:
I guess it is some sort of jump host for testing, but we don't need it in this solution

Reply:
I've deleted this for now

Reply:
Coll, also makes lots of sense!